recipes/opencv/4.x/conanfile.py

```diff
@@ -86,6 +90,8 @@ class OpenCVConan(ConanFile):
         "with_imgcodec_pfm": False,
         "with_imgcodec_pxm": False,
         "with_imgcodec_sunraster": False,
+        "with_msmf": False,
+        "with_msmf_dxva": False,
```

Comment:
these options should be removed if os is not Windows.

Reply:
Agree, I have not possibility to test on other operation system.